llvm/test/Transforms/LoopVectorize/select-cmp.ll

```diff
@@ -247,6 +247,246 @@ exit:                                     ; preds = %for.body
 }
 
 
+define i32 @select_const_i32_from_icmp_mul_use(ptr nocapture readonly %v1, ptr %v2, i64 %n) {
```

Comment:
It looks like you're already testing this in the new multicmp.ll test file. Perhaps you can just delete the test?

Reply:
Done.